core/src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -21,9 +20,6 @@ import scala.concurrent.duration._
 @InternalApi private[akka] class CassandraReadJournalConfig(config: Config, writePluginConfig: CassandraJournalConfig)
   extends NoSerializationVerificationNeeded {
 
-  private val timeBucketFormat = "yyyyMMdd'T'HH:mm"
-  private val timeBucketFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern(timeBucketFormat).withZone(ZoneOffset.UTC)
```

Comment:
The `.withZone(ZoneOffset.UTC)` was lost at the new common definition.
I don't know why that was only given at one of the places, but I guess it should be included.